kpops/cli/main.py

```diff
@@ -120,12 +122,17 @@ def setup_pipeline(
 
     handlers = setup_handlers(components_module, pipeline_config)
     return Pipeline.load_from_yaml(
-        pipeline_base_dir, pipeline_path, registry, pipeline_config, handlers
+        pipeline_base_dir,
+        pipeline_path,
+        registry,
+        pipeline_config,
+        handlers,
```

Comment:
is this a new formatting rule?

Reply:
Nop, maybe something enforced line length :thinking:

Reply:
imo for function definitions with a lot of parameters this formatting is good, but function calls take up a lot of space this way

Reply:
Ah, the whole time I thought we had [this discussion](https://github.com/bakdata/kpops/pull/363#discussion_r1338252174) here.

Reply:
What a nightmare... Had to cherry-pick commits one-by-one after the `revert`

Not sure what I did wrong, really tried to only revert the comma related stuff, but to no avail

Reply:
Should be good now tho